run-accessibility-check.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Script to run the Accessibility Check locally
+# Constants to add colour to text
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+# Build docker (can comment out if already built)
+docker-compose build
+# Test the backened

```

Comment:
May want to change this comment

Reply:
Good catch.
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/75/commits/aa4d7403c068db432054c59ee326cd08ce47687c